library/psa_util.c

```diff
@@ -330,4 +334,205 @@ mbedtls_ecp_group_id mbedtls_ecc_group_of_psa(psa_ecc_family_t curve,
 }
 #endif /* PSA_WANT_KEY_TYPE_ECC_PUBLIC_KEY */
 
+#if defined(MBEDTLS_ASN1_WRITE_C)
+/*
+ * Convert a single raw coordinate to DER ASN.1 format.
+ * Note: this function is similar to mbedtls_asn1_write_mpi(), but it doesn't
+ * depend on BIGNUM_C.
+ * Note: this function fills der_buf backward.
```

Comment:
I added that comment for some future developer looking at this function and wondering why I didn't just use `asn1_write_mpi` since it was already available. The only relevant part here is that this new function does not depend on `BIGNUM_C`. I will improve the documentation ;)